tls/s2n_cipher_preferences.c

```diff
@@ -1116,3 +1122,14 @@ int s2n_pq_kem_extension_required(const struct s2n_cipher_preferences *preferenc
     }
     S2N_ERROR(S2N_ERR_INVALID_CIPHER_PREFERENCES);
 }
+
+int s2n_cipher_preference_is_tls13_required(const struct s2n_cipher_preferences *preferences)
```

Comment:
when we use this function, we are using it to check if the cipher preferences support tls1.3.  should it be named more appropriately?

Reply:
s2n_cipher_preference_is_tls13_required -> `s2n_cipher_preference_supports_tls13(cipher_preference)`